src/gluonnlp/model/__init__.py

```diff
@@ -151,3 +152,117 @@ def get_model(name, **kwargs):
             'Model %s is not supported. Available options are\n\t%s'%(
                 name, '\n\t'.join(sorted(models.keys()))))
     return models[name](**kwargs)
+
+
+def get_tokenizer(model_name, dataset_name,
```

Comment:
Can we put this API under gluonnlp.data instead?

Reply:
Okay.